### PR TITLE
fix(partner network): interface change

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

search url updated with /members.
change GET to POST request and update the FE logic accordingly

## Why

wrong search url being used

## Issue

#980 

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [ ] I have performed [IP checks](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool) for added or updated 3rd party libraries
- [ ] I have created and linked IP issues or requested their creation by a committer
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally
- [ ] I have added tests that prove my changes work
- [ ] I have checked that new and existing tests pass locally with my changes
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by lavanya-bmw — APPROVED** [review]
lgtm

File: src/features/newPartnerNetwork/partnerNetworkApiSlice.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const apiSlice = createApi({
       BusinessPartnerRequest
     >({
       query: (body) => ({
-        url: '/legal-entities/search?page=0&size=30',
+        url: '/members/legal-entities/search?page=0&size=30',
```
Comment (evegufy) [inline src/features/newPartnerNetwork/partnerNetworkApiSlice.ts:48]: I tested it and this doesn't solve the issue... certainly also this [query](https://github.com/eclipse-tractusx/portal-frontend/blob/e5a6f82043edf5f46aa7f04a7f2f5137a4133ad0/src/features/newPartnerNetwork/partnerNetworkApiSlice.ts#L57) needs to get updated in some way - only adding /members won't be sufficient as this only returns a 404.
As you are enabled to connect to the int env, you can test this interface change very easily, please make sure to properly test fix before re-requesting the review.

**Review by evegufy — CHANGES_REQUESTED** [review]

File: src/features/newPartnerNetwork/partnerNetworkApiSlice.ts (hunk 1 above)
Comment (manojava-gk) [reply src/features/newPartnerNetwork/partnerNetworkApiSlice.ts:48]: https://business-partners.int.catena-x.net/pool/ui/swagger-ui/index.html#/Legal%20Entity%20Controller/searchLegalEntities

Look at this swagger link, only POST request has /members in the URL. GET request has not changed.

I could not test it from my end as it is giving 403(permission issue).

Comment (evegufy) [reply src/features/newPartnerNetwork/partnerNetworkApiSlice.ts:48]: As discussed today, the 403 error is why the interface change is needed in the first place, the permission issue will be solved once the interface has been adjusted.

Comment (oyo) [reply src/features/newPartnerNetwork/partnerNetworkApiSlice.ts:48]: Is that change request addressed now? Has the fix been tested?

Comment (evegufy) [reply src/features/newPartnerNetwork/partnerNetworkApiSlice.ts:48]: @oyo no, @manojava-gk is still working on the fix

**Review by evegufy — DISMISSED** [review]
As discussed yesterday, the 403 error is why the interface change is needed in the first place, the permission issue will be solved once the interface has been adjusted, @manojava-gk is working on it.

**Comment by manojava-gk** [comment]
@Phil91 Could you create IP issues?

**Comment by Phil91** [comment]
> @Phil91 Could you create IP issues?

@manojava-gk ip issues are approved, please update the dependency file :)

**Review by oyo — APPROVED** [review]

**Pull request merged**